engine/crates/integration-tests/tests/tracing/v2.rs

```diff
@@ -90,6 +91,7 @@ fn query_named() {
 }
 
 #[test]
+#[ignore]
```

Comment:
?

Reply:
ah thanks, should either remove the test or add it again. those tracing tests are quite sensible to changes and are pretty much useless for anything that involves I/O since enter&exit of spans for future concurrency are also recorded...